tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/environment/TestEnvironmentExecutor.java

```diff
@@ -50,4 +52,48 @@
      * Frees all resources for accessing/creating environment
      */
     void close();
+
+    /**
+     * Creates network for the environment.
+     */
+    void createNetwork();
+
+    /**
+     * Prepares command which will be executed inside docker container. This should be used for executing
+     * action script inside docker container. Due to permissions problems with mounted volumes script is
+     * moved to another directory.
+     * @param script The script name.
+     * @return command.
+     */
+    default String scriptExecCommand(String script) {
+        String sNewAbsolutePath = "/opt/hawkular-apm-test-local" + UUID.randomUUID();
+
+        return  "mkdir " + sNewAbsolutePath + " && " +
+                "cp " + Constants.HAWKULAR_APM_TEST_DIRECTORY + "/" + script + " " + sNewAbsolutePath + " && " +
+                "chmod +x " + sNewAbsolutePath + "/" + script  + " && " +
+                sNewAbsolutePath + "/" + script;
```

Comment:
I have no high hopes that this would work on Windows, but "/" could be replaced by `File.separator`, as in the other places.